inspirehep/modules/workflows/tasks/submission.py

```diff
@@ -148,14 +152,16 @@ def _reply_ticket(obj, eng):
                                                template,
                                                context,
                                                keep_new)
+            return {ticket_id_key: ticket_id}
```

Comment:
my point was a bit different, maybe I was not clear enough. In this step, you're not writing to the `ticket_id` key, only reading. This would mean that the decorator has to do only the skipping here if restarting, not the population of the key in `extra_data` from `persistent_data`, so it should never return a non-empty `dict`.

Reply:
I see, I was considering that myself, it makes sense